exercise/cache/points.py

```diff
@@ -335,15 +335,15 @@ def check_reveal_rule() -> None:
                         'best_submission': submission.id,
                         'points': submission.grade,
                         'formatted_points': format_points(submission.grade, True, False),
-                        'passed': (ready and submission.grade >= entry['points_to_pass']),
+                        'passed': (not unofficial and submission.grade >= entry['points_to_pass']),
```

Comment:
Is it possible that `passed` = `True` when submission.status in (REJECTED, ERROR) and `points_to_pass` = 0? Letting a rejected submission be "passed" sounds suspicious?

Reply:
No, it's not possible. REJECTED and ERROR are already filtered out before the main submission loop: https://github.com/apluslms/a-plus/blob/c9208d7d917ab86ef5b907e4d17390199e4e1235/exercise/cache/points.py#L91-L92